xml/cap_depl_admin_notes.xml

```diff
@@ -14,9 +14,38 @@
    <dm:bugtracker></dm:bugtracker>
    <dm:translation>yes</dm:translation>
   </dm:docmanager>
- </info>  
-  <para>
-      Some tips, tricks, and traps.</para>     
-<xi:include href="sec_cap_depl_admin_notes.xml"/>
-  </chapter>
+ </info>
+
+ <!-- Content entities are in repeated-content-decl.ent. Paragraph breaks and
+      othe formatting are in the entities -->
+ <para>
+      Important things to know before deploying &productname;. 
+      </para>
+
+  <sect1>
```

Comment:
indention looks odd.

Reply:
Yep. I take the lazy way out and run daps-xmlformat every so often to clean it up.

Reply:
It's usually best to make sure every commit is as well-indented as possible, otherwise the occasionally necessary `git blame` will only give you unhelpful results. (And it also bloats your repos.)